sphinx_immaterial/search_adapt.py

```diff
@@ -19,15 +19,22 @@
 class IndexBuilder(sphinx.search.IndexBuilder):
     def get_objects(
         self, fn2index: Dict[str, int]
-    ) -> Dict[str, Dict[str, Tuple[int, int, int, str, str]]]:
```

Comment:
I guess this should be a Union return type, ideally with a comment about the version.

Reply:
Added in fac934c